include/zephyr/bluetooth/services/ias.h

```diff
@@ -60,6 +68,60 @@ int bt_ias_local_alert_stop(void);
 #define BT_IAS_CB_DEFINE(_name)                                                                    \
 	static const STRUCT_SECTION_ITERABLE(bt_ias_cb, _CONCAT(bt_ias_cb_, _name))
 
+struct bt_ias_client_cb {
+	/** @brief Callback function for bt_ias_discover.
+	 *
+	 *  This callback is called when discovery procedure is complete.
+	 *
+	 *  @param conn Bluetooth connection object.
+	 *  @param err 0 on success, ATT error or negative errno otherwise
+	 */
+	void (*discover)(struct bt_conn *conn, int err);
+};
+
+/** @brief Set high alert level
+ *
+ *  @param conn Bluetooth connection object
+ *
+ *  @return Zero in case of success and error code in case of error.
+ *
+ */
+int bt_ias_client_alert_high(struct bt_conn *conn);
```

Comment:
This can return two types of error code: GATT and errno, which are both negative in the case of 

```c
	if (client_by_conn(conn)->write_handle == 0) {
		return BT_GATT_ERR(BT_ATT_ERR_INVALID_HANDLE);
	}
```

where `BT_GATT_ERR(BT_ATT_ERR_INVALID_HANDLE)` == `-EPERM`, which will be confusing. 

Suggest to return postive ATT error codes (remove `BT_GATT_ERR`), or only return errno error codes. 

This may apply multiple places

Reply:
Done